crowd-ui/src/features/home/Home-page.tsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom'
+
+
+
+const Home = () => {
+    return (
+        <div>
+            <Link to="/"><button>Image Label Verification</button></Link>
+            <Link to="/counter"><button>Counter</button></Link>
```

Comment:
should't we use 'useNavigate' instead of <link> ?

Reply:
thanks for your comment. I used Link according to react router docs.